test/config.test.ts

```diff
@@ -247,6 +247,15 @@ describe("config", () => {
             jsDoc: "extended",
         })
     );
+    it(
+        "jsdoc-dependentRequired",
+        assertSchema("jsdoc-dependentRequired", {
+            type: "Billing",
+            expose: "export",
+            topRef: true,
+            jsDoc: "extended",
```

Comment:
Does the dependent feature work when the jsDoc reader is not enabled?

Reply:
Yes, it does not rely on `jsDoc` config, shall I support it?

Reply:
I’m not sure. What do you think are arguments either way ?

Reply:
I think I should move my code to the `ExtendedAnnotationsReader` class, it is more reasonable

Reply:
Thanks. Let me know when you have revised.